src/logging.jl

```diff
@@ -2,6 +2,19 @@ import Logging, LoggingExtras
 
 export MPILogger, MPIFileLogger
 
+"""
+    default_logger()
+    default_logger(ctx::AbstractCommsContext)
+
+Return the default logger for the given context. If no context is passed, obtain the default context.
+
+Passing an MPICommsContext will return the `MPILogger`.
+Any other context will return the default `ConsoleLogger`.
+"""
+default_logger() = default_logger(context())
+default_logger(ctx::MPICommsContext) = MPIFileLogger(ctx)
```

Comment:
This seems incorrect. What is the default log dir?

Also, if this is not failing, it means that it is not tested.

Reply:
Thank you for catching this. I have added a testset for the default_logger and a test ensuring that the MPIFileLogger writes to files and stdout on root.